orchestra/models/core/models.py

```diff
@@ -220,13 +230,17 @@ class Worker(WorkerMixin, models.Model):
             The worker's Slack id if Slack integration is enabled.
         phone (str):
             The worker's phone number
+        max_autostaff_hours_per_day (float):
+            The maximum hours of work to auto-assign the worker
```

Comment:
clarify that it's computed from more than just auto-assigned work? (there's text from the doc updates we could probably copy here).

Reply:
"An upper bound on the hours of new work the Worker can be assigned per day. This is further bounded by work already completed today or already auto-assigned today."